src/lib/data-api.ts

```diff
@@ -205,7 +209,7 @@ export function createDataRecord(
   }
 
   return {
-    id: normalizePath(path + "/" + name + ".md"),
+    id: path + "/" + name + ".md",
```

Comment:
As the root folder is recorded as "/" by default, while subfolders recorded in the format of "path" rather than "/path", the includes() function of folder datasource would be affected.